### DISCUSSION TITLE
1.10.1 Issues?

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#590] [General]
Doing a test run of my mod for SI+SS, and I have had two crashes near the start of the game that seem to be related to the digital audio. On my third start now, and I am in the Knight's Test but I didn't get to bring in my backpack with the hourglass in it.

[2stderr.txt](https://github.com/user-attachments/files/17963423/2stderr.txt)
[2stdout.txt](https://github.com/user-attachments/files/17963424/2stdout.txt)
[1stderr.txt](https://github.com/user-attachments/files/17963425/1stderr.txt)
[1stdout.txt](https://github.com/user-attachments/files/17963426/1stdout.txt)

Should I switch to the latest unstable and see if I can recreate these problems?

**Comment by KnightCaptainU7** [comment]
Rat's attack now makes a spell sound?

**Comment by KnightCaptainU7** [comment]
[3stderr.txt](https://github.com/user-attachments/files/17963471/3stderr.txt)
[3stdout.txt](https://github.com/user-attachments/files/17963472/3stdout.txt)

Looks like there is a new bug introduced when a new shape is added but there is no name or valid details for it. I have custom shapes for Dupre's Armour and The Leather Vest of Flameproof, along with Paperdoll art to go with them. But if I open Dupre or Shamino, and click on them, they cause a crash.

Prior versions would just ignore the lack of name and details, no crash.

[4stdout.txt](https://github.com/user-attachments/files/17963491/4stdout.txt)
[4stderr.txt](https://github.com/user-attachments/files/17963492/4stderr.txt)

[Armous_Shapes.zip](https://github.com/user-attachments/files/17963501/Armous_Shapes.zip)
[Armour_Paperdolls.zip](https://github.com/user-attachments/files/17963502/Armour_Paperdolls.zip)

I don't have any shapes at these numbers.
```
Shape 1037 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1038 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1039 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1042 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1043 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1044 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1045 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1046 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1047 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1048 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
Shape 1051 is a monster that can be invisible, invisible flag moved from Monster info ( monster.dat ) to Actor info ( shape_info.txt ) as NPC flags.
```

**Comment by DominusExult** [comment]
> Doing a test run of my mod for SI+SS, and I have had two crashes near the start of the game that seem to be related to the digital audio. On my third start now, and I am in the Knight's Test but I didn't get to bring in my backpack with the hourglass in it.
> 
> [2stderr.txt](https://github.com/user-attachments/files/17963423/2stderr.txt) [2stdout.txt](https://github.com/user-attachments/files/17963424/2stdout.txt) [1stderr.txt](https://github.com/user-attachments/files/17963425/1stderr.txt) [1stdout.txt](https://github.com/user-attachments/files/17963426/1stdout.txt)
> 
> Should I switch to the latest unstable and see if I can recreate these problems?

definitely try whether it is in unstable as well.

If it's reliably reproducable I can test when the regression was introduced.
